poet/poet.py

```diff
@@ -40,7 +48,7 @@
   def install
 {% if resources %}
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python{{ py_version }}/site-packages"
-    %w[{{ resources|map(attribute='name')|join(' ') }}].each do |r|
+    %w[{{ resources|map(attribute='name')|map('lower')|join(' ') }}].each do |r|
```

Comment:
Should we always downcase the resource names? I think we don't need to but maybe I'm forgetting something.